src/main/java/edu/umich/tl/CourseDelete.java

```diff
@@ -1,17 +1,82 @@
 package edu.umich.tl;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Properties;
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpResponse;
+
+import edu.umich.tl.ApiCallHandler.ApiTypes;
+
 
 
 public class CourseDelete {
+	private static final String CANVAS_URL = "canvas.url";
+	private static final String CANVAS_TOKEN = "canvas.token";
+	private static final String ESB_URL = "esb.url";
 	private static Log M_log = LogFactory.getLog(CourseDelete.class);
+	protected static String canvasURL = null;
+	protected static String canvasToken = null;
+	protected static String esbURL = null;
+	//This parameter decide to make canvas or ESB api call
+	private static int typeOfApiCall;
 
 	public static void main(String[] args) {
-		M_log.debug("main(): called");
-
+		M_log.info("main(): called");
+		readPropertyFiles(args);
+		ApiCallHandler apiHandler=new ApiCallHandler();
+		apiCurrentTerm(apiHandler);
 	}
 
+	private static void readPropertyFiles(String[] args) {
+		M_log.debug("readPropertyFiles(): called");
+		Properties properties = null;
+		String propFileLocation;
+		 if(args.length == 0) {
+			 M_log.error("Command line arguments are not provided");
+			 System.exit(0);
+		 }
+		    propFileLocation = args[1];
+		    typeOfApiCall=Integer.parseInt(args[0]);
+			properties = DeleteUtils.getPropertiesFromURL(propFileLocation);
+			canvasToken = properties.getProperty(CANVAS_TOKEN);
+			canvasURL = properties.getProperty(CANVAS_URL);
+			esbURL=properties.getProperty(ESB_URL);
+	}
+	
+	private static void apiCurrentTerm(ApiCallHandler apiHandler) {
```

Comment:
I will rename the method. Yes, the method just returns the current enrollment terms.